crates/bevy_render/src/color/mod.rs

```diff
@@ -1214,48 +1368,47 @@ impl Add<Color> for Color {
 
 impl AddAssign<Vec4> for Color {
```

Comment:
We should probably remove this impl as well, for the same reasons of implicitness.